cartridges/int_algolia/cartridge/scripts/algolia/steps/sendDeltaExportProducts.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+/*
+    path to the zip:
+        Impex / src / platform / outbox / <consumer> / <exportFile> / <seqNum>.zip
+
+    zip structure
+        <seqNum>.zip / <UUID> / catalogs / <catalogID> / catalog.xml
+
+    consumer - job step parameter - comma-separated values - creates deltas for multiple consumers if requested, we'll only use one value
+    exportFile - job step parameter - it's actually a folder, the CatalogDeltaExport system job step calls it a file, so we'll keep it like this for consistencies' sake
```

Comment:
Unfortunately this is what the first job step's parameter is called (the Delta Export step) and it's set in stone as it's a platform-provided job step. Since I'm using job parameters that are shared between the two job steps, changing it would create an inconsistency between the two job steps (it wouldn't be clear that they refer to the same parameter). This is something I had to go with since I can't change it. I can show you what I mean in private if it's not clear.

Reply:
The first job parameter is called `ExportFile`, indeed, but can't you pass it any ref? e.g. `<parameter name="ExportFile" job-parameter-ref="exportDir"/>`

Reply:
In my opinion it would be more confusing if I renamed it so that we'd have the same value under two different names. If, as a customer, you'd see `exportFile` in one place and `exportDir` in another, you'd assume that they're different parameters and likely wouldn't occur to you that they might refer to the same value. Having the same name removes this ambiguity -- consistence is very important.
Also, in the end it makes sense that they named it `exportFile` since the class representing both files and folders in SFCC is `dw.io.File`.